packages/components/progress-bar/src/progress-bar.stories.ts

```diff
@@ -0,0 +1,241 @@
+  },
+  argTypes: {
+    value: {
+      control: {
+        type: 'range',
+        min: 0,
+        max: 100
+      }
+    },
+    state: {
+      control: 'inline-radio',
+      options: ['active', 'success', 'warning', 'error']
+    },
+    slot: {
+      table: {
+        disable: true
+      }
+    }
+  },
+  parameters: {
+    viewport: {
+      defaultViewport: 'default'
+    }
+  },
+  render: ({ indeterminate, state, value, label, slot }) => {
+    return html`
+      <sl-progress-bar .indeterminate=${indeterminate} .value=${value} .label=${label} .state=${state}>
+        ${slot?.() ?? html`<span>Uploaded ${value}% of 100%</span>`}
+      </sl-progress-bar>
+    `;
+  }
+} satisfies Meta<Props>;
+
+export const Basic: Story = {};
+
+export const Overflow: Story = {
+  ...Basic,
+  args: {
+    label:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. ' +
+      'Nunc lobortis mattis aliquam faucibus purus in massa tempor nec. Tincidunt eget nullam non nisi est sit amet facilisis magna. ' +
+      'Dignissim convallis aenean et tortor. Facilisis leo vel fringilla est ullamcorper. Mi proin sed libero enim sed faucibus turpis in. ' +
+      'Ullamcorper malesuada proin libero nunc consequat interdum varius. Eget egestas purus viverra accumsan in nisl nisi scelerisque. ' +
+      'Erat imperdiet sed euismod nisi porta. Pulvinar pellentesque habitant morbi tristique. Lobortis elementum nibh tellus molestie nunc non blandit massa enim.'
+  },
+  render: ({ indeterminate, state, value, label }) => {
+    setTimeout(() => {
+      const progressBar = document.querySelector('sl-progress-bar') as ProgressBar,
+        subtractButton = progressBar?.nextElementSibling,
+        addButton = subtractButton?.nextElementSibling;
+
+      addButton?.addEventListener('click', () => {
+        const newValue = Math.min(100, progressBar.value + 5);
+        progressBar.value = newValue;
+        progressBar.requestUpdate();
+      });
+
+      subtractButton?.addEventListener('click', () => {
+        const newValue = Math.max(0, progressBar.value - 5);
+        progressBar.value = newValue;
+        progressBar.requestUpdate();
+      });
+    });
+
+    return html`
+      <style>
+        sl-progress-bar {
+          margin-block-end: 32px;
+        }
+      </style>
+      <sl-progress-bar .indeterminate=${indeterminate} .value=${value} .label=${label} .state=${state}>
+        <span>Uploaded ${value}% of 100%</span>
+      </sl-progress-bar>
+      <sl-button fill="outline" class="minus">Decrease</sl-button>
+      <sl-button fill="outline" class="plus">Increase</sl-button>
+    `;
+  }
+};
+
+export const Indeterminate: Story = {
+  args: {
+    indeterminate: true,
+    label: 'Generating report',
+    slot: () => html`<span>This may take a few minutes</span>`
+  }
+};
+
+export const Download: StoryObj = {
+  args: {
+    ...Basic.args
+  },
+  argTypes: {
```

Comment:
Move the `argTypes` to the `default` export. Atm this is copied in the `All` story